vspace/src/main/java/edu/asu/diging/vspace/core/services/impl/ExhibitionManager.java

```diff
@@ -99,15 +105,15 @@ public IExhibition getStartExhibition() {
      * @throws LanguageListConfigurationNotFoundException 
      */
     @Override
-    public void updateExhibitionLanguages(Exhibition exhibition, List<String> codes, String defaultLanguage) {
+    public void updateExhibitionLanguages(Exhibition exhibition, List<String> codes, String defaultLanguage) throws ExhibitionLanguageCouldNotBeDeletedException{
```

Comment:
the exception name sounds unnecessarily long. why not `ExhibitionLanguageDeletionException`?